INSTRUCTION.md

```diff
@@ -0,0 +1,19 @@
+# Instructions to Run MySQL and App Containers
+
+
+## Running MySQL Container with Volume Attached
+```bash
+docker run -d --name mysql-container -v ~/mysql-data:/var/lib/MySQL -e MYSQL_DATABASE=app_db -e MYSQL_USER=app_user -e MYSQL_PASSWORD=1234 -e MYSQL_ROOT_PASSWORD=rootpassword -p 3306:3306 mysql-local:1.0.0

```

Comment:
Using hardcoded passwords (MYSQL_PASSWORD and MYSQL_ROOT_PASSWORD) in the Docker run command is a security risk. Consider using environment variables or Docker secrets to manage sensitive information securely.